Dockerfile

```diff
@@ -73,21 +73,47 @@ RUN \
 RUN \
 	z3_version=4.7.1 && \
 	cd /tmp && \
-	wget -O z3.zip https://github.com/Z3Prover/z3/releases/download/z3-${z3_version}/z3-${z3_version}-x64-ubuntu-16.04.zip && \
+	wget --progress=dot:mega -O z3.zip https://github.com/Z3Prover/z3/releases/download/z3-${z3_version}/z3-${z3_version}-x64-ubuntu-16.04.zip && \
 	unzip z3.zip && \
 	mv z3-${z3_version}-x64-ubuntu-16.04/bin/z3 /usr/bin/ && \
 	mv z3-${z3_version}-x64-ubuntu-16.04/bin/libz3.so /usr/bin/ && \
 	mv z3-${z3_version}-x64-ubuntu-16.04/bin/libz3java.so /usr/bin/ && \
 	rm -rf /tmp/z3-${z3_version}-x64-ubuntu-16.04 z3.zip
 
 RUN cd /tmp \
-	&& wget https://bootstrap.pypa.io/get-pip.py \
+	&& wget --progress=dot:mega https://bootstrap.pypa.io/get-pip.py \
 	&& python get-pip.py \
 	&& rm get-pip.py \
 	&& pip install mkdocs \
 	&& pip install mkdocs-cinder
 
 
+# install JetBrains JDK 8 with functional javafx
+## for headless gtk
+ENV DISPLAY :0
```

Comment:
We export this already in the startup script https://github.com/mbeddr/mbeddr.build.docker/blob/master/start.sh

Reply:
We should either remove it there or here.

Reply:
Since the readme suggests using the image directly, I'd remove it there. When developing on the Dockerfile, I used the dockerfile directly. Similarly, I'd move `JAVA_HOME` here in order to make local development reproducible.

Reply:
What do you mean by "Since the readme suggests using the image directly"? 

This image isn't meant to be used anywhere outside of a teamcity agent. Especially not to build on your local machine.

Reply:
The readme talks about `docker pull` and `docker run`, not about `start.sh`. For submitting a request like this, I tested my Dockerfile changes locally; by running docker cli commands, not `start.sh`.

Reply:
yes to run the agent, not to run any of your builds in it. That's not the goal at all of this image.